cmd/snap-update-ns/utils.go

```diff
@@ -87,6 +89,49 @@ func IsReadOnly(dirFd int, dirName string) (bool, error) {
 	return false, nil
 }
 
+// IsTrustedTmpfs returns true if a directory belongs to a tmpfs mounted by snapd.
+//
+// The function inspects the directory (represented as both an open file
+// descriptor and the absolute path) and a list of changes that were applied to
+// the mount namespace. A directory is trusted if it is a tmpfs that was
+// mounted by snap-confine or snapd-update-ns. Note that sub-directories of a
+// trusted tmpfs are not considered trusted by this function.
+func IsTrustedTmpfs(dirFd int, dirName string, changes []*Change) (bool, error) {
```

Comment:
As mentioned on IRC, I prefer if the word 'trusted' was not in the variable so adjusting the name and the comment to reflect the name change is appreciated. Perhaps isSnapdCreatedPrivateTmpfs() or mayWriteToTmpfs()?